pako/pako_manager.py

```diff
@@ -72,7 +71,20 @@ def call(self, args: list):
     def update(self):
         return self.call(self.config['update'].split()) == 0
 
-    def install_one(self, package: str, fmt: str = None) -> bool:
+    def action(self, action: str, packages: list, overrides: dict = None) -> bool:
```

Comment:
An oversight from my previous comment - we should probably mark these private with a single underscore: `_action()` and `_action_one()`